deepchem/feat/complex_featurizers/contact_fingerprints.py

```diff
@@ -228,3 +228,43 @@ def _featurize(self, complex):
       # voxels_per_edge, num_feat) so we should concatenate on the last
       # axis.
       return np.concatenate(pairwise_features, axis=-1)
+
+
+def compute_all_sybyl(mol, indices=None):
```

Comment:
I think we can use the code at https://oddt.readthedocs.io/en/latest/_modules/oddt/toolkits/extras/rdkit.html, specifically the function `_sybyl_atom_type` to infer the sybyl atom type from oddt (this code is BSD, https://github.com/oddt/oddt/blob/master/LICENSE, so I think we're fine to just copy)

Reply:
This can be done in a future PR of course!

Reply:
Sounds good! I will incorporate this in the next PR when I refactor RdkitGridFeaturizer with the reorganized functions.